### PR TITLE
remove metadata; types for credential and non-credential; add issue tags

### LINKED ISSUE #144: Add language on interop with JSON LD, making use of @vocab
https://transmute-industries.github.io/vc-json-schema/#jsonschema

```
This [term definition](https://www.w3.org/TR/json-ld/#dfn-term-definition) is https://www.w3.org/ns/credentials/issuer-dependent#JsonSchema.

JsonSchema is used for validation of W3C Verifiable Credentials, based on the latest version of the living standard.

An [issuer](https://www.w3.org/TR/did-core/#identifiers) MAY include a [credentialSchema](https://www.w3.org/TR/vc-data-model/#data-schemas) of [type](https://www.w3.org/TR/json-ld/#specifying-the-type) [JsonSchema](https://transmute-industries.github.io/vc-json-schema/#dfn-jsonschema).
```

### PR DESCRIPTION
**Pull request by decentralgabe**
fixes #144, #145

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/pull/149.html" title="Last updated on May 16, 2023, 3:15 AM UTC (3e6c30b)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-json-schema/149/dbfac75...3e6c30b.html" title="Last updated on May 16, 2023, 3:15 AM UTC (3e6c30b)">Diff</a>

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -258,83 +228,94 @@ <h3>Credential Schema</h3>
...
+                      <th>Property</th>
+                      <th>Description</th>
+                   </tr>
+                </thead>
+                <tbody>
+                   <tr>
+                      <td>id</td>
+                      <td>The constraints on the <code>id</code> property are listed in the Verifiable Credentials
+                      Data Model specification [[VC-DATA-MODEL]]. The value is expected to be a URL that identifies
+                      the credential schema associated with the <a>verifiable credential</a>.</td>
```
Comment (andresuribe87) [inline index.html:254]: I think it's important to clarify how implementers of a verifier go from "id" to the actual json_schema object that will be used to validate against.

File: index.html [hunk 2]
```diff
@@ -258,83 +228,94 @@ <h3>Credential Schema</h3>
...
+              </pre>
+            </p>
+            <p>
+              <pre class="example highlight" title="Example Email Credential Schema">
+              {
+                "@context": [
+                    "https://www.w3.org/2018/credentials/v1",
+                    "https://www.w3.org/2018/credentials/examples/v1"
+                ],
+                "id": "http://example.edu/credentials/3732",
```
Comment (andresuribe87) [inline index.html:293]: I suggest making this example consistent with the one above. For example, the `id` should be different.

File: index.html [hunk 3]
```diff
@@ -258,83 +228,94 @@ <h3>Credential Schema</h3>
...
+                   "type": "CredentialSchema2023"
+                 }
+               }
+              </pre>
+            </p>
+            <p>
+              <pre class="example highlight" title="Example Email Credential Schema">
+              {
+                "@context": [
+                    "https://www.w3.org/2018/credentials/v1",
```
Comment (andresuribe87) [inline index.html:290]: Is this for v1 or for v2?

File: index.html [hunk 4]
```diff
@@ -258,83 +228,94 @@ <h3>Credential Schema</h3>
...
+                    "https://www.w3.org/2018/credentials/examples/v1"
+                ],
+                "id": "http://example.edu/credentials/3732",
+                "type": ["VerifiableCredential", "CredentialSchema2023"],
+                "issuer": "https://example.com/issuers/14",
+                "issuanceDate": "2010-01-01T19:23:24Z",
+                "credentialSubject": {
+                  "$id": "https://example.com/schemas/email.json",
```
Comment (andresuribe87) [inline index.html:298]: Should this be `https://example.com/schemas/email-credential-schema.json` ?

File: index.html [hunk 5]
```diff
@@ -258,83 +228,94 @@ <h3>Credential Schema</h3>
...
+              {
+                "@context": [
+                    "https://www.w3.org/2018/credentials/v1",
+                    "https://www.w3.org/2018/credentials/examples/v1"
+                ],
+                "id": "http://example.edu/credentials/3732",
+                "type": ["VerifiableCredential", "CredentialSchema2023"],
+                "issuer": "https://example.com/issuers/14",
+                "issuanceDate": "2010-01-01T19:23:24Z",
+                "credentialSubject": {
```
Comment (andresuribe87) [inline index.html:297]: Some explanation on how to go from JSON schema --> CredentialSchema2023 would be useful.

File: index.html (hunk 1 above)
Comment (decentralgabe) [reply index.html:254]: there is a section on resolution and #19 to handle this

File: index.html (hunk 2 above)
Comment (decentralgabe) [reply index.html:293]: added

File: index.html (hunk 3 above)
Comment (decentralgabe) [reply index.html:290]: v2 good call

File: index.html (hunk 4 above)
Comment (decentralgabe) [reply index.html:298]: yes

File: index.html (hunk 5 above)
Comment (decentralgabe) [reply index.html:297]: see above comment

**Review by andresuribe87 — APPROVED** [review]

File: index.html [hunk 6]
```diff
@@ -140,104 +140,74 @@ <h3>Terminology</h3>
       <section class="normative">
          <h2>Data Model</h2>
          <p>
-            The following sections outlines the data model for this document.
+            The following sections outlines the data model for this document, of which there are two:
```
Comment (TallTed) [inline index.html:143]:
```suggestion
            The following sections outline the data models for this document, of which there are two:
```

**Review by TallTed — CHANGES_REQUESTED** [review]
tiny tweaks

**Pull request merged**